sys/fs/ext2fs/ext2_htree.c

```diff
@@ -300,7 +300,9 @@ ext2_htree_find_leaf(struct inode *ip, const char *name, int namelen,
 	if ((levels = rootp->h_info.h_ind_levels) > 1)
 		goto error;
 
-	entp = (struct ext2fs_htree_entry *)(((char *)&rootp->h_info) +
+	/* Preserve capability bound here. */
+	entp = (struct ext2fs_htree_entry *)(((char *)rootp) +
```

Comment:
Probably we want to use `__unbounded_addressof` here from sys/cdefs.h? This is what we generally have been using for sub-object bounds related things.